doc/content/modules/user-manual/pages/release-notes/2025.8.0.adoc

```diff
@@ -37,6 +37,25 @@ Moreover, the tooling has been improved to:
 * Find the best containment location at creation time. In this case, "a0"
 * Handle properly source/target reconnection to recompute feature chain and the best container.
 
+- Fixes the type of a `Feature` that owns `FeatureChaining`.
```

Comment:
Fixes the type of a `Feature` that owns `FeatureChaining`. => Fix an issue where the type property of a a `Feature` that owns `FeatureChaining` was not correctly set.